pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -574,22 +572,8 @@ func DeleteServiceTemplate(serviceName, serviceType, productName, isEnvTemplate,
 			_ = commonrepo.NewRenderSetColl().Update(rs)
 		}
 		// 把该服务相关的s3的数据从仓库删除
-		s3Storage, _ := s3service.FindDefaultS3()
-		if s3Storage != nil {
-			tarball := fmt.Sprintf("%s.tar.gz", serviceName)
-			file := filepath.Join(s3Storage.Subfolder, configbase.ObjectStorageServicePath(productName, serviceName), tarball)
-			forcedPathStyle := true
-			if s3Storage.Provider == setting.ProviderSourceAli {
-				forcedPathStyle = false
-			}
-			client, err := s3tool.NewClient(s3Storage.Endpoint, s3Storage.Ak, s3Storage.Sk, s3Storage.Insecure, forcedPathStyle)
-			if err != nil {
-				log.Warnf("Failed to create s3 client, err: %s", err)
-			} else {
-				if err = client.DeleteObjects(s3Storage.Bucket, []string{file}); err != nil {
-					log.Warnf("Failed to delete file %s, err: %s", file, err)
-				}
-			}
+		if err = fs.DeleteArchivedFileFromS3(serviceName, configbase.ObjectStorageServicePath(productName, serviceName), log); err != nil {
+			log.Warnf("Failed to delete file %s, err: %s", serviceName, err)
```

Comment:
why Warnf?

Reply:
see deleted line 590, we don't want to stop the process